edubots_surveys.Rnw

```diff
@@ -560,53 +560,37 @@ opinions.
 \section{Methodology}
 \label{sec:meth}
 
-Our intention from the beginning was to try to gather a sufficient amount of
-responses in all sectors, as well as cover different sectors of higher
-education. In doing so, an online survey was administered (i.e., Google Form),
-in Spanish, which was intended to be answered in a very short amount of
-time. Initially, our intention was to mirror the questions used in
-\cite{MoraChatbots2021:anon}; however, an initial exploration of results led us
-to create a second survey. The \textcolor{blue}{new survey} items focused on the use of messaging apps
+This study followed a quantitative approach towards addressing its research objectives.
+In doing so, an online survey was administered (i.e., Google Form),
+in Spanish, comprised of six questions focusing on demographic data
+(i.e., sector, gender, degrees, discipline, age, teaching experience)
+followed by four multiple choice questions allowing the participants to choose multiple answers. 
+These questions focused on the use of messaging apps
 in the teaching practice, the type of chatbot use cases that teachers consider
 useful to be used in their teaching, and the type of interactions that teachers
 prefer to have with their students while using messaging apps in their teaching.
 
-The {\bf initial survey} was done in two separate Google forms, one for
-university and other for tertiary non-university teachers. Questions and
-responses were the same, except for the type of tertiary degrees that
-\textcolor{blue}{were considered}. In both cases, we used mainly mailing lists, as well as Telegram groups,
-to spread the link to the form. The university form was spread in Spanish
-(mainly Andaluc\'{\i}a and Galicia) universities, and also Costa Rica and
-Mexico. The tertiary non-university teachers are based mainly in
-Andaluc\'{\i}a. Dissemination and answers took place in the first quarter of
+This survey was administered to both university and tertiary non-university teachers (i.e., teaching in the vocational sector). The responders were reached out through the use of mailing lists and Telegram groups. The survey was shared among university teachers that teach in 
```

Comment:
Any reason why you eliminate the fact that we were using two different Google forms?